### ISSUE TITLE
getReRequestUrl() method doesn't work as expected

### ISSUE THREAD

**Issue by hex494D49** [facebookarchive/php-graph-sdk#1071]
I'm using of Facebook PHP SDK 5.0 and this is my scenario.  I have several pages on which I am the administrator and I'd like to manage them through my application. Let's say that during the first logging process I didn't grant permissions for some of my pages. In a meantime I add some more pages  and then I decide to run logging process again in order to link some of the old (skipped) pages or the ones added later. According to Facebook PHP SDK, the method getReRequestUrl() should be a solution but it doesn't bring the permission dialog box on.  This is that snippet of code 

```
session_start();
require_once __DIR__ . '/../vendors/Facebook/autoload.php';
$fb = new Facebook\Facebook([
  'app_id' => {app_id},
  'app_secret' => {app_secret},
  'default_graph_version' => 'v3.1'
]);

$helper = $fb->getRedirectLoginHelper();
$permissions = ['manage_pages, publish_pages, read_insights'];
$loginUrl = $helper->getReRequestUrl({path-to-callback-url}, $permissions);

```
In other words, I don't see how to grant permissions for a new page. Further, I wondered if there was a way to get a response from this permission dialog box in order to know which of the permissions was given to each available pages.

**Comment by cerkiner** [comment]
Any update on this?
